src/modules/viewmodel.rs

```diff
@@ -0,0 +1,40 @@
+//! `bxt_viewmodel`
```

Comment:
```suggestion
//! `bxt_viewmodel_remove`
```

Reply:
Hmmm, I am just having the module named like that so maybe some more viewmodel stuff could be added in the future there. So you want this to be viewmodel_remove specific?

Reply:
Yes. The modules are split based on dependencies. In this case the module depends on `R_DrawViewModel` and `R_PreDrawViewModel`. If you added some more functionality here with extra dependencies, you would lose the ability to have one functionality enabled while the other functionality is disabled.